Resources/Views/Home/_drop-in-sessions.leaf

```diff
@@ -83,6 +83,13 @@
                     <p class="fs-xs mb-0">A designer, founder, and creative force behind many well-known digital products across his 20-year career. A key figure in the Delicious Generation era with projects like Disco, AppZapper, and Camino. With three Apple Design Awards and a company acquired by Facebook, Jasper brings deep experience at the intersection of design and technology. He’s currently focused on Darkroom, the most recent Apple Design Award winner, photo editor used by many. Based in Amsterdam, he’s passionate about design craft, photography, and road cycling.</a></p>
                   </div>
                 </div>
+                <div class="col d-flex align-items-center p-3">
+                  <img src="#awsImage("klemens.jpg")" width="48" alt="Klemens Strasser" class="rounded">
+                  <div class="ps-2 ms-1">
+                    <h3 class="h6 mb-0">Klemens Strasser</h3>
+                    <p class="fs-xs mb-0">Klemens is an indie developer based in Graz, Austria, who is passionate about making games and apps more accessible. Over the past decade, he has contributed to prominent apps such as Fantastical, Cardhop, and the Nuki SmartLock platform. He recently transitioned to full-time indie development to work on The Art of Fauna, a puzzle game celebrating the beauty of nature. His commitment to inclusive design earned him an Apple Design Award in 2015 as a student, as well as two Apple Design Award nominations in 2022 and 2023.</a></p>
```

Comment:
```suggestion
                    <p class="fs-xs mb-0">Klemens is an indie developer based in Graz, Austria, who is passionate about making games and apps more accessible. Over the past decade, he has contributed to prominent apps such as Fantastical, Cardhop, and the Nuki SmartLock platform. He recently transitioned to full-time indie development to work on The Art of Fauna, a puzzle game celebrating the beauty of nature. His commitment to inclusive design earned him an Apple Design Award in 2015 as a student, as well as two Apple Design Award nominations in 2022 and 2023.</p>
```

what's this closing tag doing?

Reply:
oh you copy pasted it from the one above which also has that mistake

wanna do a quick check and make sure we haven't got random tags where they shouldn't be

Reply:
Good spot 👍🏼